isotovideo

```diff
@@ -79,12 +81,17 @@ $| = 1;
 select(STDOUT);            # default
 $| = 1;
 
+
+Log::Log4perl->init($installprefix . "/etc/os-autoinst/log4perl.conf");
```

Comment:
IMHO there should be a user override, e.g. `~/.config/os-autoinst/log4perl.conf`